cmd/get/interfaces/execution_interface.go

```diff
@@ -0,0 +1,15 @@
+package interfaces
+
+import (
+	"context"
+
+	cmdCore "github.com/flyteorg/flytectl/cmd/core"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+//go:generate mockery -all -case=underscore
+
+type Fetcher interface {
```

Comment:
What's the purpose of this interface? Can you add comments? Also, why is the file called execution_interface when it also includes FetchLPVersion?

Reply:
Added the comments for it. 
Its used for exposing the create capabilities to other command modules.
eg : create exeuction which requires to fetch the launcplan for its parameters details can use this interface to fetch the data.